crates/bevy_render/src/camera/camera.rs

```diff
@@ -81,30 +143,29 @@ pub fn camera_system<T: CameraProjection + Component>(
     )>,
 ) {
     let mut changed_window_ids = Vec::new();
-    // handle resize events. latest events are handled first because we only want to resize each
-    // window once
-    for event in window_resized_events.iter().rev() {
+
+    // Collect all unique window IDs of changed windows by inspecting resized windows
+    for event in window_resized_events.iter() {
```

Comment:
This removed `.rev()` and the one below are the only actual code change, only because I think they're leftovers from a previous change, as the comments seem outdated too and we only collect all unique window IDs so the order really doesn't matter.

Reply:
My first impression is that you're correct here.